source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -95,53 +97,33 @@ local firstPersonNamePossessiveForList(listName) = {
       },
       transform: 'concatenate_list',
     },
-    {
-      transform: 'format_possessive',
-      arguments: {
-        string_to_format: {
-          source: 'previous_transform',
-        },
-      },
-    },
+    transforms.formatPossessive,
   ],
 };
 
+local personName(sameNameTransform='') = (
+  if sameNameTransform == 'contains_same_name_items' then
```

Comment:
I think Andrew comments about `contains` and `has` still applies. The strings values should also be updated as `contains_same_name_items` and `list_has_same_name_items` reads almost the same.